Source/SLKTextViewController.m

```diff
@@ -1397,7 +1400,8 @@ - (void)slk_willShowOrHideKeyboard:(NSNotification *)notification
     
     // Begin and end frames are the same when the keyboard is shown during navigation controller's push animation.
     // The animation happens in window coordinates (slides from right to left) but doesn't in the view controller's view coordinates.
-    if (!CGRectEqualToRect(beginFrame, endFrame))
+    // Second condition: check if the height of the keyboard changed.
+    if (!CGRectEqualToRect(beginFrame, endFrame) || fabs(preKeyboardHeight - self.keyboardHC.constant) >= FLT_EPSILON)
```

Comment:
Just out of my habit when dealing with floating point numbers. :stuck_out_tongue:
As you mentioned, a direct comparison will do the work and there should not affect anything at UI level.